src/modules/NativeTokenLimitModule.sol

```diff
@@ -116,6 +116,10 @@ contract NativeTokenLimitModule is BaseModule, IExecutionHookModule, IValidation
         override
     {} // solhint-disable-line no-empty-blocks
 
+    function preSignatureValidationHook(uint32, address, bytes32, bytes calldata) external pure override {
+        return;
+    }
+
```

Comment:
Nit: do we want to follow empty return or empty function body like above? I don't feel strongly either way but we should be consistent.

Reply:
Good call, I think we should update these to just be empty function bodies (and add the linter warning bypass).